e2e/factories/fixtures/playwright.ts

```
@@ -0,0 +1,39 @@
+ * Extended Playwright test context with data factory integration.
+ * Provides automatic cleanup.
+ *
+ * Usage:
+ * - test({factories}) - Full factory access
+ */
+export const test = base.extend<{
+    factories: FactoryList;
+    factoryManager: FactoryManager;
+    stats: Record<string, number>
+}>({
+    factories: async ({}, use) => {
```

Biome (2.1.2)

[error] Unexpected empty object pattern.



(lint/correctness/noEmptyPattern)